orca_detector/train.py

```diff
@@ -80,4 +89,18 @@ def run(**params):
 
 
 if __name__ == '__main__':
-    run()
+    # parse command line parameters and flags
+    parser = argparse.ArgumentParser(description='OrcaDetector - W251 (Summer 2019)',
+                        epilog='by Spyros Garyfallos, Ram Iyer, Mike Winton')
```

Comment:
nit: move these values to a single variable in params

Reply:
will do

Reply:
done